enterprise/templates/enterprise/base.html

```diff
@@ -46,6 +45,6 @@ <h1>
     {% endblock %}
 
     {% block contents %}{% endblock %}
-    <script type="text/javascript" src="{% static 'common/js/vendor/bootstrap.bundle.js' %}"></script>
+    <!-- <script type="text/javascript" src="{% static 'common/js/vendor/bootstrap.bundle.js' %}"></script> -->
```

Comment:
any reason to remove bootstrap here but keep commented?

Reply:
not sure if other pages that use this template expect bootstrap? would double check